src/main/java/org/takes/http/BkBasic.java

```diff
@@ -54,10 +54,35 @@
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  * @checkstyle IndentationCheck (500 lines)
  */
+@SuppressWarnings("PMD.LongVariable")
 @EqualsAndHashCode(of = "take")
 public final class BkBasic implements Back {
 
     /**
+     * The http header "X-Takes-LocalAddress".
+     */
+    public static final String HEADER_TAKES_LOCAL_ADDRESS =
```

Comment:
@yegor256 I think I need some confirmation that I've got the idea.So we need a class Header with 4 static final instances (for ex. public static final Header XTakesLocalAddress = new Header("...") with toString() returning the required value) and use the Header.XTakesLocalAddress.toString() where we need in the code?
From another point of view, why not make the Header as an "enum" instead of a class?
So what should be a better solution?